### ISSUE TITLE
Lots of wireless redstone causes crash

### ISSUE THREAD

**Issue by T3charmy** [licks92/WirelessRedstone#63]
When the server crashes, the traceback goes to Wireless redstone. then after that it says something about SQL, so I figured alright well must be a problem loading all the channels, which makes sense. So, I switched to setting to use SQL off, and it converted everything to YAML.. Converted everything fine... but then I noticed... it seems to load the files on main thread instead of creating a new one, which then causes the server to crash...

TL;DR: WR loads all the signs on main thread, which causes main thread to lock up, and crash when there is a lot of WR to load.
